app/celery/scheduled_tasks.py

```diff
@@ -0,0 +1,94 @@
+from datetime import datetime, timedelta
+
+from flask import current_app
+from sqlalchemy.exc import SQLAlchemyError
+
+from app import notify_celery
+from app.clients import STATISTICS_FAILURE
+from app.dao.invited_user_dao import delete_invitations_created_more_than_two_days_ago
+from app.dao.notifications_dao import delete_notifications_created_more_than_a_week_ago, get_notifications, \
+    update_notification_status_by_id
+from app.dao.users_dao import delete_codes_older_created_more_than_a_day_ago
+
+
+@notify_celery.task(name="delete-verify-codes")
+def delete_verify_codes():
+    try:
+        start = datetime.utcnow()
+        deleted = delete_codes_older_created_more_than_a_day_ago()
+        current_app.logger.info(
+            "Delete job started {} finished {} deleted {} verify codes".format(start, datetime.utcnow(), deleted)
+        )
+    except SQLAlchemyError:
+        current_app.logger.info("Failed to delete verify codes")
```

Comment:
do we really need these log statements? They'll be one line above a stack trace that will include "delete_verify_codes" in it, right?